File: src/main/java/com/acn/vinlotteri/model/Lottery.java
package com.acn.vinlotteri.model;


import lombok.Builder;
import lombok.Data;
import org.springframework.data.annotation.CreatedDate;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.LocalDateTime;

@Builder
@Data
@Entity
public class Lottery {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    @CreatedDate
    private LocalDateTime createdDate;
    private Boolean open;
    private Long ticketCost;
    private String phoneNumber;


}

File: src/static/api/Lottery.php
<?php

class Lottery
{
    public $id;
    public $createdDate;
    public $isOpen;
    public $ticketCost;
    public $phoneNumber;

    public function __construct($id, $createdDate, $isOpen, $ticketCost, $phoneNumber)
    {
        $this->id = $id;
        $this->createdDate = $createdDate;
        $this->isOpen = $isOpen;
        $this->ticketCost = $ticketCost;
        $this->phoneNumber = $phoneNumber;
    }
}